lib/b2c/sessions.ts

```diff
@@ -505,6 +505,52 @@ export interface SessionsRevokeResponse {
   status_code: number;
 }
 
+// MANUAL(authenticateJwt)(TYPES)
+
+// Request type for `sessions.authenticateJwt`
+export interface SessionsAuthenticateJwtRequest {
+  /**
+   * The JWT to authenticate. You may provide a JWT that has expired according to its `exp` claim and needs
+   * to be refreshed. If the signature is valid and the underlying session is still active then Stytch will
+   * return a new JWT.
+   */
+  session_jwt: string;
+
+  /**
+   * If set, remote verification will be forced if the JWT was issued at (based on the "iat" claim) more than that many seconds ago.
+   * If explicitly set to zero, all tokens will be considered too old, even if they are otherwise valid.
+   */
+  max_token_age_seconds?: number;
+}
+
+// Request type for `sessions.authenticateJwtLocal`
+export interface SessionsAuthenticateJwtLocalRequest {
+  /**
+   * The JWT to authenticate. The JWT must not be expired in order for this request to succeed.
+   */
+  session_jwt: string;
+
+  /**
+   * The maximum allowable difference when comparing timestamps.
+   * It defaults to zero.
+   */
+  clock_tolerance_seconds?: number;
+
+  /**
+   * If set, return an error if the JWT was issued (based on the "iat" claim) more than max_token_age_seconds seconds ago.
+   * If explicitly set to zero, all tokens will be considered too old, even if they are otherwise valid.
+   */
+  max_token_age_seconds?: number;
+
+  /**
+   * The value used to compare timestamp claims ("exp", "nbf", "iat"). It
+   * defaults to the current date (new Date()).
+   */
```

Comment:
same as the other file:
```suggestion
  /**
   * The value used to compare timestamp claims ("exp", "nbf", "iat").
   * It defaults to the current date (new Date()).
   */
```